dashboard/app/models/contact_rollups_raw.rb

```diff
@@ -33,6 +88,36 @@ def self.extract_parent_emails
     ContactRollupsV2.execute_query_in_transaction(query)
   end
 
+  def self.extract_scripts_taught
+    source_sql = <<~SQL
+      SELECT email, scripts.name, MAX(se.updated_at) AS updated_at
+      FROM users u
+      JOIN sections se ON se.user_id = u.id
+      JOIN scripts ON scripts.id = se.script_id
+      WHERE scripts.name in (#{CSF_SCRIPT_LIST})
```

Comment:
This is a long list of string, which is expensive for string comparison. 
Can we just get the script name first, then later in the processing step we can decide if that script belongs to CSF category (or other categories that we may care about later).

Reply:
Ok, I thought we had landed on doing filtering in the extraction step. I'll remove it though.

Reply:
Yeah, I thought that were to filter out data that we obviously don't need. In this specific case, I think script_name and course_name are valuable. We can use them to create more categories than just CSF.